packages-exp/auth-exp/src/core/auth/register.ts

```diff
@@ -70,6 +73,20 @@ export function registerAuth(clientPlatform: ClientPlatform): void {
       ComponentType.PUBLIC
     )
   );
+
+  _registerComponent(
+    new Component(
+      _AUTH_INTERNAL_COMPONENT_NAME,
+      container => {
+        const auth = _castAuth(
+          container.getProvider('auth-exp').getImmediate()!
```

Comment:
here's that auth-exp again, we should put it in a variable

Reply:
If only it was already defined in a variable...

Done